### PR TITLE
feat: add health summary to topology card

### PR DESCRIPTION
**Pull request by Prashanth-Nelli**

### CODE REVIEW

**Comment by moshloop** [comment]
@Prashanth-Nelli -  What you have added is a component summary, not a health summary which is only for checks that are linked to components

**Comment by moshloop** [comment]
It doesn't seem to consistently display the correct values:

Screenshot 2022-10-16 at 10 22 37
Screenshot 2022-10-16 at 10 22 31
Screenshot 2022-10-16 at 10 22 02

File: src/components/HealthChecksSummary/index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  if (!checks?.length) {
+    return null;
+  }
+
+  return (
+    <div className="flex pb-2">
+      <Link className="flex cursor-pointer" to="/health">
+        <p className="flex items-center text-xs">Health Checks:</p>
+        <p className="flex items-center text-xs ml-1">
+          <Chip
```
Comment (moshloop) [inline src/components/HealthChecksSummary/index.tsx:48]: We don't want to show chips with 0 values

**Comment by moshloop** [comment]
@Prashanth-Nelli please add a heart icon

**Comment by Prashanth-Nelli** [comment]
closing this PR as these changes are covered in https://github.com/flanksource/flanksource-ui/pull/553

**Pull request closed without merging**